src/styles/navbar.module.css

```
@@ -0,0 +1,29 @@
+@import "./palette.css";
+@import url("https://fonts.googleapis.com/css?family=Roboto&display=swap");
+
+.logo {
+  width: 10rem;
+  height: 2rem;
```

```suggestion
  height: 1.4rem;
```
make the logo a bit smaller